tests/dummy/app/routes/nodes/detail/draft-registrations.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+import permissions from 'ember-osf/const/permissions';
+
+export default Ember.Route.extend({
+    model() {
+        let node = this.modelFor('nodes.detail');
+        let drafts = node.get('draftRegistrations');
+        return Ember.RSVP.hash({
```

Comment:
Not a blocking comment for this PR, but...

Conventionally, Ember favors using nested resources, and it gets super-snarky if the model hook depends on two items- so using RSVP.hash in a model hook is gently disfavored.

There are a few alternative ways to handle this (like `setupController` or `afterModel`) depending on your use case.